src/main/java/com/tabnine/statusBar/TabnineStatusBarWidget.java

```diff
@@ -149,12 +139,15 @@ private void update() {
               }
               final StateResponse stateResponse = getStateResponse();
               final ServiceLevel serviceLevel = getServiceLevel(stateResponse);
-              final Map<String, RestartStatus> globalRestartStatus =
-                  getGlobalRestartStatus(stateResponse);
               final Icon icon = getIcon(serviceLevel);
               this.component.setIcon(icon);
 
-              updateText(serviceLevel, globalRestartStatus);
+              if (serviceLevel == ServiceLevel.PRO || serviceLevel == ServiceLevel.BUSINESS) {
```

Comment:
I'd keep the `updateText` function and leave this peace of code there instead of inline-ing the function